nbs/examples/Large_Datasets_Tutorial.ipynb

```diff
@@ -0,0 +1,726 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    train.to_parquet('./data', partition_cols=['unique_id'], index=False)
</pre><p>Please define a temporary directory here and use its name</p>

<pre class="ql-syntax" spellcheck="false">tmpdir = tempfile.TemporaryDirectory()
train.to_parquet(tmpdir.name, partition_cols=['unique_id'], index=False)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1074/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>